scripts/experienceView.js

```diff
@@ -0,0 +1,92 @@
+var articleView = {};
+
+articleView.handleNav = function() {
+  $('.top-nav').on('click', 'li.tab', function(){
+    var tabClickContent = $(this).data('content');
+    if (tabClickContent === 'home') {
+      $('#experiences').hide();
```

Comment:
Not technically wrong, but see if you can convert these from IDs to data attributes, then hide them all and select the one to show with the tabClickContent variable.